.github/workflows/test-cassandra.yml

```diff
@@ -0,0 +1,34 @@
+on:
+    push:
+        branches:
+            - master
+            - main
+        paths:
+            - 'cassandra/**'
+    pull_request:
+        paths:
+            - 'cassandra/**'
+name: "Tests Cassandra"
+jobs:
+    Tests:
+        runs-on: ubuntu-latest
+        services:
+            mongo:
+                image: 'cassandra:latest'
+                ports:
+                    - '27017:27017'
```

Comment:
This port is the mongodb one, which could be causing the failure in the Test Suite